.config/forest.dic

```diff
@@ -13,8 +13,10 @@ callee
 CBOR
 CID/M
 CIDs
+cli
```

Comment:
It would be nice to use only one form of this. I'm fine with either `cli` or `CLI`. Unless there's a reason for this?

Reply:
It was a module name I think - the backticks tip was really important, thank you :) I'll add it to a README